src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -440,6 +442,15 @@ string ValidAssemblerString (string s)
 			}
 		}
 
+		bool ShouldIgnoreSplitConfigs ()
+		{
+			if (String.IsNullOrEmpty (CustomBundleConfigFile)) {
```

Comment:
Its not just the presense of a custom config file , its a specific setting (the "abi" one) which controls the split configs in this case.

Reply:
Ignore this. I miss read the if statement

Reply:
Yes, but it can be specified only in the JSON file? Or is there a way to configure it using some other means?

Reply:
Odd, I've only now seen your "ignore this" comment, ignore my reply then :)